### DISCUSSION TITLE
Feat. Create Label Categories and Merge Label and Tags

### DISCUSSION THREAD

**Discussion by rolymaz** [chatwoot/chatwoot#8553] [Feature Requests]
Request 1:
Being able to organise Labels (or tags) into categories would be very helpful in summarizing conversations. Our inboxes receive a range of conversation categories.   I would like to run a report or query that provides a list of all conversations by Label Category, then Label.   
E.g. 
Label Category: Customer Service
Label: > Order enquiry
Label: > Product enquiry

Label Category: Technical Support
Label: > Faulty connection
Label: > Unable to login

Request 2: 
Can we 'merge' tags and labels? 
Tags are applied to conversations but it would be great if the label was applied instead. By 'merging' tags and labels, we could simplify the database schema - which would make reporting easier.

**Comment by matheus-caldeira** [comment]
PR: https://github.com/chatwoot/chatwoot/pull/13114
 
This PR introduces a hierarchical label system that allows labels to have parent-child relationships. This enhancement enables better organization and categorization of conversations through nested label structures (e.g., Department > Team > Sub-team), providing more granular control over conversation management and reporting.

![Hierarchical Labels UI](https://github.com/user-attachments/assets/0cd537b4-5666-4c45-a07c-49d04b44212e)
